tests/test_cli.py

```
@@ -334,6 +334,36 @@ def test_cli_setup_command_with_duplicate_sections(mocker):
     mock_setup_run.assert_called_once_with(sections=["firmware", "android"])
 
 
+def test_cli_setup_command_with_update_available(mocker):
+    """Test the 'setup' command when an update is available."""
+    mocker.patch("sys.argv", ["fetchtastic", "setup"])
+    mock_setup_run = mocker.patch("fetchtastic.setup_config.run_setup")
+    mock_get_upgrade_command = mocker.patch(
```

Ruff (0.14.2)

Local variable `mock_get_upgrade_command` is assigned to but never used

Remove assignment to unused variable `mock_get_upgrade_command`

(F841)